### ISSUE TITLE
ResponseHandler - missing 1 required positional argument: 'loop'

### ISSUE THREAD

**Issue by thanosexcite** [pnuckowski/aioresponses#113]
Hi there. After an upgrade of aiohttp to version 3.5.0, I'm getting the following exception when running tests using aioresponses 0.5.0:

```
/usr/local/lib/python3.6/asyncio/coroutines.py:110: in __next__
    return self.gen.send(None)
/usr/local/lib/python3.6/site-packages/aioresponses/core.py:256: in _request_mock
    response = await self.match(method, url)
/usr/local/lib/python3.6/asyncio/coroutines.py:110: in __next__
    return self.gen.send(None)
/usr/local/lib/python3.6/site-packages/aioresponses/core.py:232: in match
    response = await matcher.build_response(url)
/usr/local/lib/python3.6/asyncio/coroutines.py:110: in __next__
    return self.gen.send(None)
/usr/local/lib/python3.6/site-packages/aioresponses/core.py:105: in build_response
    resp.content = stream_reader_factory()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def stream_reader_factory():
>       protocol = ResponseHandler()
E       TypeError: __init__() missing 1 required positional argument: 'loop'

/usr/local/lib/python3.6/site-packages/aioresponses/compat.py:23: TypeError
```

Looks like in aiottp 3.5.0, with this change https://github.com/aio-libs/aiohttp/pull/3372 , the loop parameter is now required.

Thanks

**Comment by akalex** [comment]
@Derfirm Is there any estimation on when the PR #112 supposed to be merged?

**Comment by Derfirm** [comment]
@akalex Unfortunately I have no idea when PR can to be merged =/

**Comment by pnuckowski** [comment]
Released new version 0.5.1 :)
